src/app/services/folder.js

```diff
@@ -747,6 +750,13 @@ module.exports = (Model, App) => {
     return { folders, last: limit > rawFolders.length };
   };
 
+  const updateFolderLastModification = async (folderId) => {
+    const folder = await Model.folder.findOne({ where: { id: folderId } });
```

Comment:
update using where instead of doing a select and then an update, is faster.

Reply:
The update was not working, seems that `updated_at` cannot be updated without the set because the property is "managed" by Sequelize. `updatedAt` does not exist on `FolderAttributes`, they exist because on the model `timestamps` is set to true https://github.com/internxt/drive-server/blob/14605cf1615869eab7d3cb40c0c70d3650eeb6d2/src/app/models/folder.ts#L3-L12